src/kademlia/dht_tracker.cpp

```diff
@@ -155,9 +163,10 @@ namespace libtorrent { namespace dht
 		m_refresh_timer.expires_from_now(seconds(5), ec);
 		m_refresh_timer.async_wait(std::bind(&dht_tracker::refresh_timeout, self(), _1));
 
-		m_dht.bootstrap(m_state.nodes, f);
+		// bootstrap with mix of IP protocols via want/nodes/nodes6
+		m_dht.bootstrap(concat(m_state.nodes, m_state.nodes6), f);
```

Comment:
no, they're separate. and actually, perhaps it would make sense to not insert nodes with the ``flag_no_id`` flag set in a sorted location, but to just append to the end.